source/backend/dal/Security/Permissions.cs

```diff
@@ -123,16 +123,19 @@ public enum Permissions
         [Display(GroupName = "document", Name = "document-admin", Description = "Can perform admin functions on documents.")]
         DocumentAdmin = 39,
 
+        [Display(GroupName = "document", Name = "generate-documents", Description = "Can generate documents.")]
+        GenerateDocuments = 40,
```

Comment:
Also, we need one for an Admin to add template documents.

Reply:
I think for the story for that one I just said that simply checking for the admin role is good enough for adding template documents. I don't see a case where another user type would be given access to that page.

Reply:
Do you mean what is stated in the story? It mentions that the user has the role of Admin. There is no mention of the claims. For synchronization with Mayan, there is document-admin permission that we could use if we want to group both admin responsibilities together.

Reply:
yep, My idea is that we wouldn't use a claim at all and just use role membership.